src/NovelImageRect.h

```diff
@@ -38,6 +39,7 @@ class NovelImageRect : public NovelRenderObject {
   void configureObjectBuffers() final;
 
 private:
+  NovelLoggingService _console;
```

Comment:
Why does `ImageRect` need to hold onto the logging service?

Reply:
Currently `ImageRect` performs file-handling for loading images - I'd gladly take this away (the hold on the logging service), but I'm not sure how we would go about referencing it to call the log in `NovelImageRect.cpp` since it doesn't onto a service that already holds it.

Reply:
IIRC, I believe we had discussed this in Discord and found that we would need to really overhaul `NovelRunner` to support this which would be OoS atm. May I resolve this?

Reply:
Created Issue for this - closing.